### ISSUE TITLE
how to test one img？

### ISSUE THREAD

**Issue by czy5638222** [mala-lab/InCTRL#21]
How should I write the code to input an image and return the anomaly score? Please help me

**Comment by baraa-m** [comment]
@Diana1026 
Thank you for sharing this Github, it's very interesting. 
I have exactly the same question above, I hope someone can help. The idea is to test the pretrained model on one image and see either the mask of image if there is an anomaly or "no anomaly detected" ...
Best regards!

**Comment by Diana1026** [comment]
Hi, I did not write any script for testing one single image, so maybe you can modify the code by yourself. Besides, our paper did not involve the anomaly segmentation task as we did not use the mask ground truth during training. However, I believe you can obtain a simple anomaly similarity map based on interpolating the holistic map.
